src/dfx/src/commands/language_service.rs

```diff
@@ -40,7 +44,35 @@ pub fn exec(env: &dyn Environment, opts: LanguageServiceOpts) -> DfxResult {
             .get_defaults()
             .get_build()
             .get_packtool();
-        let package_arguments = package_arguments::load(env.get_cache().as_ref(), packtool)?;
+
+        let mut package_arguments = package_arguments::load(env.get_cache().as_ref(), packtool)?;
+
+        // Include actor alias flags
+        let canister_names = config
+            .get_config()
+            .get_canister_names_with_dependencies(None)?;
+        let agent_env = create_agent_environment(env, None /* opts.network */)?;
+        let canister_id_store = CanisterIdStore::for_env(&agent_env)?;
+        for canister_name in canister_names.into_iter() {
+            match Principal::from_text(canister_name.clone())
+                .or_else(|_| canister_id_store.get(&canister_name[..]))
+            {
+                Ok(canister_id) => package_arguments.append(&mut vec![
+                    "--actor-alias".to_owned(),
+                    canister_name,
+                    Principal::to_text(&canister_id),
+                ]),
+                Err(_) => (),
+            };
+        }
```

Comment:
- no need for `.into_iter()` in the for loop
- `get_canister_names_with_dependencies` returns canister names, not principal IDs, so we don't need to try `Principal::from_text`
- `&canister` covers it, no need for `[..]`, unless I am missing something
- addresses lint warning `error: you seem to be trying to use 'match' for destructuring a single pattern. Consider using 'if let'`

```suggestion
        for canister_name in canister_names {
            if let Ok(canister_id) = canister_id_store.get(&canister_name)
            {
                package_arguments.append(&mut vec![
                    "--actor-alias".to_owned(),
                    canister_name,
                    Principal::to_text(&canister_id),
                ])
            };
        }
```

Reply:
Thanks, looks good. It's been a few years since I've used Rust (and it's my first time in the DFX repo), so I appreciate the suggestions.